### PR TITLE
Convert utils to ES6

### PR DESCRIPTION
**Pull request by jdlehman**
- move utils to `src/utils`
- convert to ES6
- resolves Issue #171

### CODE REVIEW

File: components/AltNativeContainer.js [hunk 1]
```diff
@@ -7,7 +7,6 @@ var React = require('react-native')
 var mixinContainer = require('./mixinContainer')
 var assign = require('object-assign')
 
-var View = React.View
```
Comment (goatslacker) [inline components/AltNativeContainer.js]: ?

Comment (goatslacker) [inline components/AltNativeContainer.js]: Oh I see what's going on. This is broken :( we need to pass in View into this.altRender and React.DOM.div for the other one.

File: utils/ActionListeners.js [hunk 2]
```diff
@@ -1,62 +0,0 @@
-'use strict'
```
Comment (goatslacker) [inline utils/ActionListeners.js]: I was hoping we'd keep these here so we don't break linking:

`require('alt/utils/Util')`

vs

`require('alt/lib/utils/Util')`

Comment (goatslacker) [inline utils/ActionListeners.js]: The utils folder would need to be .gitignored though.

**Comment by goatslacker** [comment]
This is awesome, thanks :+1: 

I converted some of the utils from class form to prototypes because they take up way less code that way.

**Comment by jdlehman** [comment]
Cool by me!

**Pull request merged**